recipes/xarray-safe-rcm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xarray-safe-rcm" %}
+{% set version = "2023.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-safe-rcm-{{ version }}.tar.gz
+  sha256: 98b8ac60e8caf8bf911fb7805f94ed0d93110121257389c233b900c764437153
+
+build:
+  skip: true  # [py<310]
```

Comment:
this has been generated by `grayskull`. Is this truly necessary, especially since this is as `noarch: python` package?

Reply:
the bot says it is actually not allowed, so let's remove it:
```suggestion
```